tests/src/main/python/rest/tests/MeasureParametersAPITests.py

```diff
@@ -0,0 +1,31 @@
+import sys
+import os
+from sys import path
+from os import getcwd
+from utilities.TestClass import TestClass
+path.append(getcwd() + "/../../src/main/python/rest/tests/extract")
+import swagger_client
+from swagger_client.api.fhir_measures_api import FHIRMeasuresApi
+
+class MeasureParametersAPITest(TestClass):
+
+    def test_simpleErrorCase(self):
+        fhirMeasuresApi = FHIRMeasuresApi(swagger_client.ApiClient(self.configuration))
+        result = None
+        try:
+            resp = fhirMeasuresApi.get_measure_parameters('2021-03-15','https://fhir-internal.dev:9443/fhir-server/api/v4','default','badMeasureID')
```

Comment:
I'd like to see the FHIR server endpoint come from some type of external property instead of being hard-coded.

Reply:
Good point, even the tenant could be established somewhere are reused instead of hard coded. Wondering what other common values we may want to consider.